src/components/Footer/Footer.tsx

```diff
@@ -26,18 +26,22 @@ const Footer = () => (
             <p
               className="
                 font-bold mb-4 flex text-center justify-center md:justify-start
-            " 
+            "
             >
               Powered by
             </p>
-            <img src={RTLogo} alt="Rotherham Titans Rugby" className="w-16 mx-auto md:mx-0" />
+            <a href="https://www.titans-rugby.com/">
```

Comment:
Would it be worth using target="_blank" here so that clicking on the links opens into a new tab?

Reply:
Yes, good shout, will change it :)